### DISCUSSION TITLE
Rudder Stack integration

### DISCUSSION THREAD

**Discussion by jalberto** [plausible/analytics#228] [Anything Else]
## Precheck

Please note that this tracker is only for feature requests. Do not use the issue tracker for help or support. [Our docs](https://docs.plausible.io/) are a great place for most answers, but if you can’t find your answer there, you can [contact us](https://plausible.io/contact). Thanks!

## Prerequisites
- [x] I have searched open and closed issues to make sure that the feature has not yet been requested.

##Feature request

I will love to see Rudder (https://rudderstack.com/) integration for plausible. Both products are opensource and will be a great combination.

**Comment by flexa-david** [comment]
Any progress with this? This would be great

**Reply by metmarkosaric** [reply]
thanks for the feedback! no short terms plans that i can tell you about at the moment. we're a small team with limited development resources so need to take it one step at a time. there are some other more frequently requested features that we are working on. most of [our integrations](https://plausible.io/docs/integration-guides) to date have been done by community members so perhaps someone else has something in mind for this...

**Comment by ukutaht** [comment]
Pinging @psrikanth88 as the biggest [contributor on RudderStack](https://github.com/rudderlabs/rudder-server/graphs/contributors). What would it take to integrate Plausible with Rudderstack? I *think* it would have to be some module on your side that sends events to [our API](https://plausible.io/docs/events-api)?
